docs/DeveloperGuide.md

```diff
@@ -366,10 +366,17 @@ This feature allows user to view patient's details as PatientTable does not show
 This feature can be accessed using the `view` command, which currently only support the viewing of patient which follows
 the following format `view t/patient PATIENT_ID` where `PATIENT_ID` is a valid id of a patient.
 
-Given below is the sequence diagram when a user provides an example of a valid `view` command
-(`view t/patient P001`) to see the complete details of the patient.
+Given below is an activity diagram showing the event flow when the user wants to view a patient's details:
 
-![ViewPatientCommandCreation](images/ViewPatientCreation.png)
+![ViewPatientActivityDiagram](images/ViewPatientActivityDiagram.png)
```

Comment:
btw I think your activity diagram is a bit inaccurate, to be exact the user never needs to input twice right ?

Reply:
Since your diagram mentions `User enters the command...` twice, one is for the command, one is for the id (in fact we only input once)

Reply:
resolved